src/providers/ObsEditProvider.js

```diff
@@ -64,30 +63,20 @@ const ObsEditProvider = ( { children }: Props ): Node => {
   };
 
   const createObsPhotos = useCallback( async photos => Promise.all(
-    photos.map( async photo => ObservationPhoto.new( photo?.image?.uri ) )
+    photos.map( async photo => ObservationPhoto.new( photo?.image?.uri, undefined, true ) )
   ), [] );
 
   const createObservationFromGalleryPhoto = useCallback( async photo => {
-    const originalPhotoUri = photo?.image?.uri;
```

Comment:
this code should not be removed -- we want to get exif data using these methods.